File: mario-more/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    int h;
    //Checking if input number is in allowed range
    do
    {
        h = get_int("Height: \n");
    }
    while (h >= 9 || h < 1);

    printf("\n");
    //First For loop prints columns (Height)
    for (int i = 0; i < h; i++)
    {
        //Prints spaces in each column
        for (int j = 0; j < h - i - 1; j++)
        {
            printf(" ");
        }
        //Left side
        for (int y = 0; y < i + 1; y++)
        {
            printf("#");
        }
        //Space in the middle
        printf("  ");
        //Right side
        for (int k = 0; k <= i; k++)
        {
            printf("#");
        }
        //Prints a new row after each iteration
        printf("\n");
    }


}

File: sentimental-mario-more/mario.py
import cs50

# Declaring main functioin


def main():
    # Calling functions inside main
    height = get_pyramid_height()
    print_pyramid(height)

# Making sure we prompt until we get int in range


def get_pyramid_height():
    while True:
        try:
            h = cs50.get_int("Height: ")
            if h > 8 or h < 1:
                print("Enter a number between 1 and 8")
            else:
                return h
        except ValueError:
            print("Not an interger number")

# Printing pyramid using for loop


def print_pyramid(h):
    for i in range(h):
        # Spaces at the begining
        for j in range(h - i - 1):
            print(" ", end="")
        # Left side
        for y in range(i + 1):
            print("#", end="")
        # Space in the middle
        print("  ", end="")
        # Right side
        for k in range(i + 1):
            print("#", end="")
        print()


main()
